operator/build/multi-arch.Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM quay.io/mew2057/ansible-operator:latest
+# Multi-arch build for IBM Spectrum Scale CSI Operator
+# usage: docker buildx build --platform=linux/amd64 -t my_image_tag .
+
+FROM quay.io/mew2057/ansible-operator:$TARGETARCH
```

Comment:
At some point we're going to want to  move this out of my personal quay repo,  but this shouldn't hold back the PR.

Reply:
Yeah, I agree - hopefully, your upstream issue gets some movement.